### DISCUSSION TITLE
Please support RTL in components

### DISCUSSION THREAD

**Discussion by mmtaheri** [mdbootstrap/TW-Elements#2265] [Share ideas & request features]
Moved child div in state RTL Body in component INPUT
please support RTL in component

**Comment by juujisai** [comment]
Hello @mmtaheri ! We already have a task for this milestone to check RTL issues inside our components and start fixing them! I hope we can improve the RTL styling really soon!

Link to the issue:
https://github.com/mdbootstrap/TW-Elements/issues/2077

Best Regards!
